resumegenerator.html

```diff
@@ -283,20 +283,20 @@ <h5 class="text-center my-2">A rough preview. Differs from the LaTeX output. <br
                                     <!-- <div class="linkedin p-0">
                                         <img src="images/linkedin.png" class="rounded mx-0.5 d-block p-0" style="width: 20px;" id="gitT">
                                     </div> -->
-                                    <div class="git p-0">
-                                        <a id="githubLink" href="#">
-                                            <img src="images/git.png" class="rounded mx-0.5 d-block p-0"
-                                                style="width: 20px; margin-right: 2px;">
-                                        </a>
-                                    </div>
 
-                                    <div class="linkedin p-0">
-                                        <a id="linkedinLink" href="#">
-                                            <img src="images/linkedin.png" class="rounded mx-0.5 d-block p-0"
-                                                style="width: 20px;">
-                                        </a>
+                                    <!-- changed icons to text  -->
+                                    <div class="linkedin p-0">  
+                                        <a id="linkedinLink" href="#" class="text-black no-underline hover:text-gray-500">Linkedin</a>
+                                    </div> 
+                                    <span class="mx-2">|</span>
+                                    <div class="git p-0">
+                                        <a id="githubLink" href="#" class="text-black no-underline hover:text-gray-500">Github</a>
                                     </div>
-
+                                    <span class="mx-2">|</span>
+                                    <div class="website p-0">  
+                                        <a id="websiteLink" href="#" class="text-black no-underline hover:text-gray-500">Linkedin</a>
```

Comment:
```suggestion
                                        <a id="websiteLink" href="#" class="text-black no-underline hover:text-gray-500">Website</a>
```

Reply:
Ok let me check i corrected that I'll check and push the corrected code

Reply:
####corrected the links text and added redirect functionality #22 

fixed : #24 
all the links are working as expected 

"Linkedin | Github | Website "